### ISSUE TITLE
`$PATH` not inherited when opening WSL Code window via cmd line

### ISSUE THREAD

**Issue by StanzillaManticore** [microsoft/vscode-remote-release#6457]
- VSCode Version: 1.65.2
- Local OS Version: Windows 11 21H2
- Remote OS Version: Linux STAN-PC 5.10.60.1-microsoft-standard-WSL2 #1 SMP Wed Aug 25 23:20:18 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
- Remote Extension/Connection Type: WSL
- Logs:

Steps to Reproduce:

1. Open a folder via `C:\Windows\sysnative\wsl.exe code .`
2. Try to run a program that is on your $PATH

Does this issue occur when you try this locally?: No
Does this issue occur when you try this locally and all extensions are disabled?: Yes

**Comment by bamurtaugh** [comment]
To clarify, the program is installed in WSL (not just Windows)? What is the program?

> when opening WSL Code window via cmd line

Does the program work when opening a Remote-WSL window from within VS Code, rather than an external terminal?

**Comment by StanzillaManticore** [comment]
> To clarify, the program is installed in WSL (not just Windows)? What is the program?
Only in WSL, and anything on path, like python etc.
> > when opening WSL Code window via cmd line

> Does the program work when opening a Remote-WSL window from within VS Code, rather than an external terminal?
yes

**Comment by bamurtaugh** [comment]
Could you expand on why you're using the command `C:\Windows\sysnative\wsl.exe code .`? Are you able to open VS Code from a WSL terminal directly, i.e.:

- Open WSL terminal for your WSL distro
- Find the folder / directory you want to open in VS Code, `cd` to it
- Run `code .` or `code-insiders .`
- VS Code (or VS Code Insiders) opens with that folder open with Remote-WSL

**Comment by bamurtaugh** [comment]
When you open projects in VS Code from Fork without using Remote-WSL, is your path inherited? Trying to discern if this is specific to remote development, or moving to VS Code from Fork in general.

**Comment by StanzillaManticore** [comment]
> When you open projects in VS Code from Fork without using Remote-WSL, is your path inherited? Trying to discern if this is specific to remote development, or moving to VS Code from Fork in general.

Yeah it is, also I forgot to say, this used to work at some point, sadly I don't remember when exactly it broke.

**Comment by bamurtaugh** [comment]
@aeschli have there been any changes in how Remote-WSL handles pathing?

**Comment by aeschli** [comment]
No changes. 

The steps are still very unclear. Please @StanzillaManticore give more detailed steps, maybe with screenshots so we don't need to guess.

- you open a remote WSL window using `wsl.exe code .`
- you start a program. How? From the integrated terminal of the VS Code remote window?
- program is not found. What does `env | grep PATH` show?

**Comment by StanzillaManticore** [comment]
Okay sorry, let's do this properly:

- Install [Fork](https://fork.dev/)
- Open Fork, go to "Preferences - Custom Commands"
- Add a new custom command with these options
	
![image](https://user-images.githubusercontent.com/58569561/159735410-cebe50a3-5e19-4f91-881f-86e9b6bf1ccb.png)

- Go to "File - Open Repository" and open a git repo that is inside of a WSL 2 container
- Go to "Open in" and select the new custom command you made 
	
![image](https://user-images.githubusercontent.com/58569561/159735598-b096568d-f5f2-4bf7-8778-a2fc8d7267fd.png)

- Try to use a cli app

![image](https://user-images.githubusercontent.com/58569561/159736818-df9a81dc-5667-4316-881e-c04549108722.png)


Now that I look at that again, it might also just be that it is not using my zsh profile?
